fe/fe-core/src/main/java/com/starrocks/journal/bdbje/BDBEnvironment.java

```diff
@@ -347,7 +370,14 @@ public List<Long> getDatabaseNames() {
                 names = replicatedEnvironment.getDatabaseNames();
                 break;
             } catch (InsufficientLogException e) {
```

Comment:
1. What's the difference between InsufficientLogException and RollbackException?
2. Why does the pull request call setup() when encountering the InsufficientLogException?
3. Does it will throw InsufficientLogException and RollbackException if I remove some logs for only one FE?

Reply:
1. InsufficientLogException and RollbackException are two unrelated exceptions.
2. I was in this PR by the way to fix the problem of processing InsufficientLogException. As I described in the 2 paragraphs of this pr message.
3. If the current FE has extra log, it will throws RollbackException. If the log is too far behind, it will throws InsufficientLogException.